helpers/class.Utils.php

```diff
@@ -180,7 +180,7 @@ static public function buildAssessmentItemRefsTestMap(XmlDocument $test, taoQtiT
         $basePath = helpers_File::truePath($basePath);
         $basePath .= DIRECTORY_SEPARATOR;
         
-        $documentURI = preg_replace('/^file:\/{3}/', '', $test->getDomDocument()->documentURI);
+        $documentURI = preg_replace("|^file:/+|", '', $test->getDomDocument()->documentURI);
```

Comment:
Can you replace the + by {1,3} to not match more than 3 / ? 
Can you escape the / to keep the pattern clear ?
Can you keep the / to delimit pattern to keep consistency over the project ?